docs/resources/rediscloud_flexible_subscription.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: "rediscloud"
+page_title: "Redis Cloud: rediscloud_flexible_subscription"
+description: |-
+  Flexible Subscription resource in the Terraform provider Redis Cloud.
+---
+
+# Resource: rediscloud_flexible_subscription
+
+Creates a Flexible Subscription within your Redis Enterprise Cloud Account.
+This resource is responsible for creating and managing subscriptions.
+
+~> **Note:** The payment_method property is ignored after Subscription creation.
+
+~> **Note:** The creation_plan block allows the API server to create a well-optimised infrastructure for your databases in the cluster.
+The attributes inside the block are used by the provider to create initial
+databases. Those databases will be deleted after provisioning a new
+subscription, then the databases defined as separate resources will be attached to
+the subscription. The creation_plan block can ONLY be used for provisioning new
+subscriptions, the block will be ignored if you make any further changes or try importing the resource (e.g. `terraform import` ...).
+
+## Example Usage
+
+```hcl
+data "rediscloud_payment_method" "card" {
+  card_type = "Visa"
+}
+resource "rediscloud_subscription" "subscription-resource" {
```

Comment:
shouldn't the resource now be called `rediscloud_flexible_subscription` in the example?